locales/en/breaches.ftl

```diff
@@ -67,8 +67,10 @@ breach-checklist-link-mozilla-vpn = { -brand-mozilla-vpn }
 
 ## Prompts the user for changes when there is a breach detected of password
 
-# { $breachedCompanyLink } will link to the website of the company where the breach occurred
 breach-checklist-pw-header-2 = Go to the company’s website to change your password and enable two-factor authentication (2FA).
+# { $breachedCompanyLink } will link to the website of the company where the breach occurred
+breach-checklist-pw-header-3 = Go to { $breachedCompanyLink } to change your password and enable two-factor authentication (2FA).
```

Comment:
```suggestion
breach-checklist-pw-header-link = Go to { $breachedCompanyLink } to change your password and enable two-factor authentication (2FA).
```

Let's not use 3, since this is not versioned string anymore.